src/features/DisplayControls/DrugChart/components/TimeCell.jsx

```diff
@@ -2,22 +2,57 @@ import React from "react";
 import PropTypes from "prop-types";
 import "../styles/TimeCell.scss";
 import SVGIcon from "./SVGIcon.jsx";
+import { TooltipCarbon } from "bahmni-carbon-ui";
+import NoteIcon from "../../../../icons/note.svg";
+import { ifMedicationNotesPresent } from "../utils/DrugChartUtils";
 
 export default function TimeCell(props) {
-  const { minutes, status, administrationInfo } = props;
+  const {
+    minutes,
+    status,
+    administrationInfo,
+    medicationNotes,
+    doHighlightCell,
+    highlightedCell,
+  } = props;
   let left, right;
+    
+
+  const toolTipContent = (
```

Comment:
The variable is also not required, you replace the usages with medicationNotes directly